src/oscar/templates/oscar/dashboard/catalogue/product_update.html

```diff
@@ -305,6 +306,24 @@ <h3>{% trans "Upload, change or remove images" %}</h3>
                                 {% endblock recommended_products_content %}
                             </div>
                         {% endblock recommended_products %}
+
+                        {% block seo %}
+                            <div class="tab-pane" id="seo">
+                                <div class="table-header">
+                                    <h3>{% trans "Search Engine Optimization" %}</h3>
+                                </div>
+                                <div class="well product-seo-details">
```

Comment:
`.product-seo-details` isn't being used anywhere it seems - if not then we don't need the class.